tests/scattering1d/test_numpy_scattering1d.py

```diff
@@ -101,6 +101,20 @@ def test_Scattering1D_filter_factory_T(self, backend):
                     N, J, Q, T, filterbank)
                 assert (phi_f['sigma'] == 0.1/T)
 
+    def test_Scattering1D_average_global(self, backend):
```

Comment:
let's do this for the other backends as well since there's the new `sum` primitive

Reply:
you were right, the other backends didn't work as intended !
thanks ...

i have written a fix + more tests in [bbab64b](https://github.com/kymatio/kymatio/pull/919/commits/bbab64b274152586938f7054e3a447d49aacea55)